core/src/test/java/brooklyn/entity/drivers/ReflectiveEntityDriverFactoryTest.java

```diff
@@ -54,12 +58,22 @@ protected void assertDriverIs(Class<?> clazz) {
         assertTrue(driver.getClass().equals(clazz), "driver="+driver+"; should be "+clazz);
     }
     
+    protected void assertDriverIs(Class<?> clazz, Location location) {
```

Comment:
@robertgmoss I agree. I didn't wanted to include many changes on the rest of tests but that's cleaner.